doc/usermanual/darkroom/modules/color/lut3d.xml

```diff
@@ -0,0 +1,94 @@
+		%darktable_dtd;
+		]>
+<sect3 status="draft" id="lut3d">
+
+  <title>3D LUT</title>
+
+  <indexterm>
+    <primary>modules</primary>
+    <secondary>lut3d</secondary>
+  </indexterm>
+
+  <indexterm>
+    <primary>lut3d</primary>
+  </indexterm>
+
+  <sect4>
+
+    <title>Overview</title>
+
+    <informaltable frame="none">
+      <tgroup cols="2" colsep="0" rowsep="0">
+        <colspec colwidth="6*"/>
+        <colspec colwidth="4*"/>
+        <tbody>
+          <row>
+            <entry>
+              <para>
+                A <emphasis>3D LUT</emphasis> is a tridimensional table which allows to
+                transform any RGB value into another RGB value, normaly used for film
+                simulation and color grading.
+              </para>
+              <para>
+                The module accepts <emphasis>.cube</emphasis> and <emphasis>.png (haldclut)</emphasis> files.
+                The 3D LUT data are not saved in database nor in the XMP file, but just the
+                path inside the 3D LUT folder (see below).
+              </para>
+            </entry>
+            <entry>
+              <graphic fileref="darkroom/modules/images/lut3d-1.png" scalefit="1" width="80%" align="center" />
+            </entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </informaltable>
+
+    <para>
+      The module lut3d needs to find the 3D LUT file at the same place in your 3D LUT folder
+      to calculate the output image.
+      This means you have to backup properly your 3D LUT folder.
+      Sharing an image with its XMP is useless if the recipient has not the same 3D LUT file
+      in his 3D LUT folder.
+    </para>
+
+  </sect4>
+
+  <sect4>
+
+    <title>Usage</title>
+
+    <sect5>
+      <title>file selection</title>
+      <para>
+        File selection is inactive as long as the 3D LUT folder (where you have stored your LUT files)
+        is not defined in <emphasis>3D LUT root folder</emphasis> under
+        <emphasis>preferences/core options/miscellaneous</emphasis>.
+      </para>
+    </sect5>
+
+    <sect5>
+      <title>application color space</title>
+      <para>
+        A 3D LUT is relative to a specific color space. You have to select the one for which it has been built.
+        Cube files are usually related to <emphasis>REC.709</emphasis> while most of others are
+        related to <emphasis>sRGB</emphasis>.
+      </para>
+    </sect5>
+
+    <sect5>
+      <title>interpolation</title>
+      <para>
+        The interpolation method defines the way to calculate colors which are not exactly on a node
+        of the RGB cube (described by the 3D LUT).
+        There are three interpolation methods available: <emphasis>tetrahedral</emphasis> (the default one),
+        <emphasis>trilinear</emphasis> and <emphasis>pyramid</emphasis>.
+        Usually you don't see the difference between the interpolation methods except with small size LUTs.
```

Comment:
"The difference between the interpolation methods are not noticeable, except for small size LUTs". Maybe. I don't like this sentence either, but it's the best I have right now.

I would add something about what the compromise is - as far as interpolation algorithms go, there's often a "this one gives better results but it's super slow" - is that the case here? Or is it just a matter of "this one looks better in some cases, and this one in some other cases"?

Reply:
I haven't noticed any calculation time difference. 
When color differences are visible that's probably because the lut is too small and interpolation gives visible shift. Then that's a matter of taste.